eigentrust-zk/src/ecdsa/native.rs

```diff
@@ -28,6 +28,34 @@ where
 	big_to_fe(x_big)
 }
 
+#[derive(Copy, Clone, Default, Debug, Eq, PartialEq, PartialOrd, Ord)]
+/// Recovery id struct
+pub struct RecoveryId(u8);
+
+impl RecoveryId {
+	/// Create a new [`RecoveryId`] from the following 1-bit argument:
+	///
+	/// - `is_y_odd`: is the affine y-coordinate of 𝑘×𝑮 odd?
```

Comment:
Is this still a question?

Reply:
This comment explains what `is_y_odd` is, using a question 😄 
`is the affine y-coordinate of 𝑘×𝑮 odd?` yes(true), no(false)

Reply:
Oh! Nice, I get it.